Eatery Watch App Extension/Queries/campusEateries.graphql

```diff
@@ -49,6 +49,19 @@ query CampusEateries {
                 }
             }
         }
+        expandedMenu {
+        category,
```

Comment:
remove the comma for consistency? haven't worked with graphql so unsure if its necessary